spec/controllers/api/v1/recipes_controller_spec.rb

```diff
@@ -25,57 +25,117 @@
   end
 
   describe '[POST] #create' do
-    let(:recipe_params) do
-      {
-        recipe: {
-          name: 'Leczo',
-          content: 'Very good dish'
-        }
-      }
+    let(:call) do
+      post :create, params: { recipe: { name: name, content: content, level: level } }
     end
-
+    let(:name) { 'Leczo' }
+    let(:content) { 'Very good dish' }
```

Comment:
```suggestion
    let(:content) { 'Very good meal' }
```
Dish would rather mean the plate itself not a meal.